model/resource/host.yaml

```diff
@@ -70,3 +70,51 @@ groups:
           The version string of the VM image or host OS as defined in
           [Version Attributes](README.md#version-attributes).
         examples: ['0.1']
+  - id: host.cpu
+    prefix: host.cpu
+    type: resource
+    brief: >
+      A host's CPU information
+    attributes:
+      - id: vendor_id
+        requirement_level: opt_in
+        type: string
+        brief: >
+          Processor manufacturer. First part of the CPUID identifier - a maximum 12-character name of the manufacturer.
```

Comment:
Should we have a fixed list of vendor IDs? Similar to what we have for e.g. `host.arch`

Reply:
That's a good idea but from what I can see the list of known processor manufacturer ID strings can be quite big. For example I find a list of vendors at https://en.m.wikipedia.org/wiki/CPUID#EAX=0:_Highest_Function_Parameter_and_Manufacturer_ID. I guess it's a slightly more "flexible" field than the `arch` one, right?

Reply:
I guess since these are fixed this is fine we can mark this as solved. 

My aim was to make it the language a bit more understandable i.e. using Intel instead of "GenuineIntel" or AMD instead of "AuthenticAMD". But it seems like it is common enough to 'fake' the CPUID that we can just leave the value without 'interpreting' it.